spec/acceptance/v1/departments_spec.rb

```diff
@@ -48,4 +51,56 @@
       end
     end
   end
+
+  get 'departments/:department_id/categories' do
```

Comment:
update this as per the route changes done in routes.rb

Reply:
It is already consistent with routes.rb, all the test cases pass

For your reference I am providing the route I got using rails routes:
department_categories GET    /departments/:department_id/categories(.:format)                                                  api/v1/departments#show_categories

Reply:
shall I add the / before departments there?